### ISSUE TITLE
Power Button in AudioLink Controller is Global. It should be Local instead.

### ISSUE THREAD

**Issue by BluWizard10** [llealloo/audiolink#323]
**Describe the bug in detail:**
Power Toggle in the AudioLink Controller is Global, when it shouldn't be.

**Provide steps/code to reproduce the bug:**
Use the current Prefab. Using thw Powe Button on Teeh's AudioLink Controller is applied to everyone.

**Expected behavior:**
The Toggle should be Local!

**Additional Information:**
It's annoying when some people wanna see AudioLink, and those who don't for some reason like to Toggle it off from the Controller. It's annoying because it's synchronized across the entire instance instead of the local user.

Why is this? Why can't the Toggle on the Controller be Local only?

**Comment by float3** [comment]
the core maintainers are currently on vacation until early november, we will try to address it then

**Comment by fundale** [comment]
> **Describe the bug in detail:** Power Toggle in the AudioLink Controller is Global, when it shouldn't be.
> 
> **Provide steps/code to reproduce the bug:** Use the current Prefab. Using the Power Button on Teeh's AudioLink Controller is applied to everyone. v2.0.0.
> 
> **Expected behavior:** The Toggle should be Local!
> 
> **Additional Information:** It's annoying when some people wanna see AudioLink, and those who don't for some reason like to Toggle it off from the Controller. It's annoying because it's synchronized across the entire instance instead of the local user.
> 
> Why is this? Why can't the Toggle on the Controller be Local only?

Using Controller Sync Mode? Or just as a default setting?
Appears to function with the current "vanilla" controller (when using Controller Sync Mode)
Or is this something specific to Teeh's Controller? (I am unfamiliar with what you're referring to as Teeh's Controller)

**Comment by BluWizard10** [comment]
My bad, I was referring to the newer AudioLink Controller that Teeh helped design. It's the one with the modernized UI.

**Comment by fundale** [comment]
Is this with the Controller Sync Mode setting set to "ExcludePower" on the Controller script?

**Comment by BluWizard10** [comment]
> Is this with the Controller Sync Mode setting set to "ExcludePower" on the Controller script?

It's the new one, yeah.

**Comment by BluWizard10** [comment]
So it seems that the Power Button has `Global Toggle` attached to it.

This should be REMOVED.

![image](https://github.com/user-attachments/assets/6ab743de-e7a7-4864-a63e-f18dc873e37c)

**Comment by pema99** [comment]
Fundale is not taking about the new control design. They are talking about a feature added in the most recent release, which lets you control how it is synced. One of the modes on the drop-down let's you sync everything except the power button. Are you using it? I tested it for last release and it seemed to work fine.

**Comment by fundale** [comment]
The Controller script has a setting that disabled that script on runtime locally, and it is not synced
Each client also checks to see if it's script is enabled before taking any commands to enable or disable AudioLink

![image](https://github.com/user-attachments/assets/b1e899e0-21dd-4af0-8822-5a79216a4b26)

**Comment by BluWizard10** [comment]
> The Controller script has a setting that disabled that script on runtime locally, and it is not synced
> 
> Each client also checks to see if it's script is enabled before taking any commands to enable or disable AudioLink
> 
> 
> 
> ![image](https://github.com/user-attachments/assets/b1e899e0-21dd-4af0-8822-5a79216a4b26)
> 
> 
> 
> 

This is a really terrible decision in my honest opinion. I think choosing to sync the AudioLink Power state (effects off/on) that affects everyone in the instance, should be removed entirely. ONLY should we allow the Audio Controls to be synced in the instance, no matter what.

**Comment by pema99** [comment]
I disagree, that's just removing functionality from the user for no reason. We could change the default setting, though.

Pretty much all of the settings on the controller can have adverse effect on everyone in the instance, power and gain are just the most obvious. If we decide that default is bad, the only default that makes sense is no sync imo.

**Comment by BluWizard10** [comment]
> I disagree, that's just removing functionality from the user for no reason. We could change the default setting, though.
> 
> 
> 
> Pretty much all of the settings on the controller can have adverse effect on everyone in the instance, power and gain are just the most obvious. If we decide that default is bad, the only default that makes sense is no sync imo.

What I'm really disagreeing on is how the Power (effects on/off) is being globally synced by default. I want that changed, make the default be Local always.

That way, World Creators do not have to have their own Toggles for it.
